vulnerable_people_form/integrations/notifications.py

```diff
@@ -42,7 +42,7 @@ def send_message(registration_number, app=current_app):
     message = {
         "submission_id": registration_number,
         "nhs_number": form_answers()["nhs_number"],
-        "date_of_birth": f"{date_of_birth['year']}/{date_of_birth['month']}/{date_of_birth['day']}",
+        "date_of_birth": f"{date_of_birth['year']}/{date_of_birth['month']:0>2}/{date_of_birth['day']:0>2}",
```

Comment:
i would create a method out of this and add it to form_utils so you can add a few extra tests around it to make sure it works as expected for different examples

Reply:
Created a method. Do the tests look ok?